nemo/collections/nlp/parts/peft_config.py

```diff
@@ -60,10 +60,14 @@ def __init__(self, cfg):
         else:
             kv_channels = cfg.kv_channels
         projection_size = kv_channels * cfg.num_attention_heads
+        num_query_groups = cfg.get("num_query_groups", None)
+        if num_query_groups is None:
+            num_query_groups = cfg.num_attention_heads
+        qkv_projection_size = projection_size + (2 * kv_channels * num_query_groups)
```

Comment:
will this continue to work for non-group query attention models?

Reply:
Yes! Tested with llama2 7b as well.